docs/core/additional-tools/vscode-dotnet-runtime.md

```diff
@@ -0,0 +1,30 @@
+---
+title: .NET Install Tool for Extension Authors
```

Comment:
```suggestion
title: .NET Install Tool for VSCode Extension Authors
```

Not sure how these docs will be surfaced but sprinkling "VSCode" in front of extension authors felt more applicable since there's lots of types of extensions for various platforms.